ckanext/fjelltopp_theme/assets/css/fjelltopp-theme.scss

```diff
@@ -26,7 +26,7 @@ body {
     border-bottom: none;
 }
 
-.main, .homepage .hero {
+.main, .homepage .hero, .dashboard-organisation-main {
```

Comment:
Why does organisation need to be treated differently to the other tabs? Surely there should be one rule that styles all four tabs?